### DISCUSSION TITLE
"Error: lexical error: invalid char in json text."

### DISCUSSION THREAD

**Discussion by bonushenricus** [eblondel/zen4R#81] [Q&A] [answered]
I have prepared my record
`myrec <- ZenodoRecord$new()
myrec$setTitle("VINDICTA project Practice 3")
myrec$setAccessRight("open")
myrec$setUploadType("image")
myrec$setImageType("photo")
myrec$addCreator(name = "Gabrielli, Enrico",affiliation = "Stuard")
myrec$setKeywords(c(
"Animalia","Arthropoda","Insecta","Hemiptera","Pentatomidae"
))
myrec$setDescription("ENGA2022060401")`

but when I try do deposit

`myrec <- zenodo$depositRecord(myrec)`
there was an error

> Error: lexical error: invalid char in json text.
                                       <!DOCTYPE HTML PUBLIC "-//W3C//
                     (right here) ------^

Thank you

**Comment by eblondel** [comment]
can you share your `sessionInfo`, need to know the package version. There was an issue similar to this, in previous version that was fixed.

**Comment by bonushenricus** [comment]
Hi and thank you
I attach the session info
[r_session_info.txt](https://github.com/eblondel/zen4R/files/9041402/r_session_info.txt)

**Comment by bonushenricus** [comment]
Ok I try in RStudio with version 4.2, and it doesn't work.
[r_session_info.txt](https://github.com/eblondel/zen4R/files/9041505/r_session_info.txt)
I try also with ' to replace " quotas. It doesn't work.

**Comment by eblondel** [comment]
@bonushenricus I tried the same on Zenodo in prod and sandbox. I can't reproduce the issue. Which infra do you use? https://zenodo.org or https://sandbox.zenodo.org ? Did you make sure to specify a token that was created with deposit rights?

**Comment by bonushenricus** [comment, marked as answer]
Hi Emanuel
you must excuse me
I didn't realize that I hadn't verified the email for my zenodo account!
Now works!

**Reply by eblondel** [reply]
Ok i've converted this issue into discussion, so people can find the answer, it's not the first time it occurs.

**Reply by bonushenricus** [reply]
very good idea!
